pkg/ddl/table.go

```diff
@@ -228,6 +235,15 @@ func (w *worker) onRecoverTable(jobCtx *jobContext, job *model.Job) (ver int64,
 		tableInfo := tblInfo.Clone()
 		tableInfo.State = model.StatePublic
 		tableInfo.UpdateTS = metaMut.StartTS
+
```

Comment:
Why we move these code from `recoverTable` to here? Seems like only the second schemaDiff would use `tids` previously.